src/commands/cmd_tdigest.cc

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+#include "command_parser.h"
+#include "commander.h"
+#include "server/redis_reply.h"
+#include "server/server.h"
+#include "status.h"
+#include "string_util.h"
+#include "types/redis_tdigest.h"
+
+namespace redis {
+namespace {
+constexpr auto kCompressionArg = "compression";
+constexpr auto kErrWrongKeyword = "T-Digest: wrong keyword";
```

Comment:
We could remove the prefix(`T-Digest`) in the error message to be consistent with other commands. Users should know the context according to the command they sent, so that we can avoid creating a dedicated error message for each command.

Reply:
Hi @git-hulk ,

Got it! Thanks for your suggestions! 😊

I have removed all `T-Digest:` prefixes from error message.

Best Regards,
Edward